src/main/java/racingcar/service/RacingCarService.java

```diff
@@ -0,0 +1,57 @@
+package racingcar.service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import racingcar.model.Car;
+import racingcar.utlis.Convertor;
+import racingcar.validator.NameValidator;
+
+public class RacingCarService {
+
+	List<Car> cars = new ArrayList<>();
+
+	public void playRound() {
+		cars.forEach(Car::decideMove);
+	}
+
+	public void saveCars(String carsNames) throws Exception {
+		List<String> carNameList = new ArrayList<>(Arrays.asList(Convertor.separateNamesByDelimiter(carsNames)));
+		NameValidator.checkNameList(carNameList);
+		save(carNameList);
+	}
+
+	public void save(List<String> carNameList) throws Exception {
+		for (String name : carNameList) {
+			cars.add(new Car(name));
+		}
+	}
+
+	public int findMaxPosition() {
+		int maxPosition = 0;
+		for (Car car : cars) {
+			if (car.isOverMaxPosition(maxPosition)) {
+				maxPosition = car.getPosition();
+			}
+		}
+		return maxPosition;
+	}
+
+	public List<String> findWinner() {
+		int maxPosition = findMaxPosition();
+		return cars.stream()
+			.filter(car -> car.isWinner(maxPosition))
+			.map(Car::getName)
+			.collect(Collectors.toList());
+	}
+
+	public List<Car> findAllCars() {
+		return cars;
```

Comment:
외부에서 의도치 않게 자동차 목록을 변경할 수 있겠네요.
변경이 불가능 하도록 아래와 같이 변경하면 어떨까요?

```suggestion
		return Collections.unmodifiableList(cars);
```